climada_petals/hazard/tc_tracks_forecast.py

```diff
@@ -92,7 +92,11 @@ class TCForecast(TCTracks):
     data : list of xarray.Dataset
         Same as in parent class, adding the following attributes
         - ensemble_member (int)
-        - is_ensemble (bool; if False, the simulation is a high resolution deterministic run
+        - is_ensemble (bool; if False, the simulation is a high resolution
+                       deterministic run)
+        - forecast_time (numpy.datetime64): timepoint of the initialisation of
+            the numerical weather prediction run
```

Comment:
yeah i think the parameter name could be better here. `forecast_initialisation` or `initialisation_time` would be clearer (or 'initialization' depending on our spelling policy).

usually this reflects the time that the forecast calculations were started. so a forecast time of 26/02/2022 00:00 will mean it was started at 00:00, published a few hours later, and the forecast itself will start at 00:00 and run for 240 hours in 6 hour steps (though these vary by provider).

again, this is off the top of my head so correct me if i've misremembered @ThomasRoosli @manniepmkam

Reply:
Is the name of the attribute confusing or the description of the attribute? The only thing I am proposing in this pull request is the description of the previously existing attribute, because it was missing. I tried to be as precise as possible in the description. I think if the user is not familiar with those terms (they should be), then a search in the relevant documentation of e.g. ECMWF will reveal the meaning.
If we want to change the name of that attribute we could also consider `run_datetime` to be consistent with the Forecast class. What do you think @manniepmkam @chahank?

Reply:
@ChrisFairless reply is correct, `forecast_time` is the time when the forecast simulation is initiated. For me both `initialisation_time` and `run_datetime` are good to me if `forecast_time` is a confusing name.

Reply:
For me the issue is that both the naming can be confusing, and the docstring is not clear enough. No big deal, but it can easily be improved.

Reply:
consistency with other classes is seldom a bad idea - should we change to `run_datetime`?

Reply:
I agree with changing the name of the attribute to `run_datetime`.
I suggest to keep the docstring as it is. It mentions "initialization" and "numerical weather prediction". A user that is not familiar with the topic might do a internet search and hopefully ends up at the combination of those two terms on wikipedia: "https://en.wikipedia.org/wiki/Numerical_weather_prediction#Initialization". I assume it is not the idea to link to the wikipedia articles in our docstrings. I am very happy for further suggestions to improve the docstring.